Encountify/ViewModels/BaseViewModel.cs

```diff
@@ -64,6 +65,7 @@ public List<Location> DummyLocationList()
                 new Location("Sereikiškių parko Bernardinų sodas", latitude : 54.68413305498285, longitude : 25.29522580235671, category : (int) Category.Park ),
                 new Location("Lietuvos nacionalinis dailės muziejus", latitude : 54.68130476957157, longitude : 25.289818468853266, category : (int) Category.Museum ),
                 new Location("Užupio angelas", latitude : 54.68035, longitude : 25.29515, category : (int) Category.Monument),
+                new Location("Pavadinimas", latitude : 54.68, longitude : 25.295, category : (int) Category.SportFacility)
```

Comment:
Maybe it's only on my phone (my testing phone is a little bit old) but the locations from Dummy List are returned without the '.', only locations that are shown correctly for me are from the database. 
Please tell me if it is only the bug on my phone, thanks

Reply:
After some changes I tested on the phone Android 7.1 - API 25 and it seems to work fine